content/200-concepts/150-data-platform/08-billing/03-invoices.mdx

```diff
@@ -0,0 +1,75 @@
+- In July, we bill you on the first and issue an invoice with the full amount of the `Scale` plan
+
+![invoicing diagram](pdp-monthly-billing.png)
+
+## Invoices with overages and overage refunds
+
+Paid plans have an unlimited number of team members, environments, and usage of the Prisma Data Proxy, but **they do have quotas**. The quotas limit how many members and environments, and how much Data Proxy usage you can have within the price of the plan.
+
+When you **exceed any of these quotas, you are incurring overages** which we bill at the fixed price in the plan. If you remove the overage (only applies to team members and environments), we include a refund in the invoice for the time during which the overage was not applicable.
+
+This section explains how overages are billed and refunded for unused team members and environments in your invoice. The examples are based on the `Scale` plan.
+
+### Overages on team members or environments
+
+The `Scale` plan has quotas of 5 team members and 5 environments. An example of an overage is when you add an additional 6th team member or a 6th environment to your project.
+
+- When you add team members or environments that become an overage, we do not bill you immediately.
+- The pro-rated amount for the overage until the end of the current month is added to the invoice for the following month
+- A full amount of the overage is also added for the invoice for the following month
+
+Here is an example:
+
+- For June, you pay the invoice for the full month of June on the 1st
+- On June 5th, you add a 6th team member
+- The pro-rated (Jun 5 - Jun 30) charge for the 6th team member overage is added to the July invoice
+- The charge for the 6th team member overage for the month of July is also added to the July invoice
+
+![team members overage diagram](pdp-invoice-overage-team-member.png)
+
+### Refunds for removed overages of team members or environments
+
+If you go back to being within your project plan quota by removing a team member that was in overage, we immediately refund you the amount for the time during which the additional team member will not be in use until the end of the month.
+
+![team members refund diagram](pdp-invoice-refund-team-member.png)
+
+### Overages on Data Proxy usage
+
+The `Scale` plan has a quota of 30 million CPU milliseconds. Overages begin when you exceed the 30 million CPU milliseconds. The overages are billed at the fixed price of the `Scale` plan.
+
+![Data Proxy usage overages diagram](pdp-invoice-data-proxy-overages.png)
+
+## Tracking overages on the Data Proxy usage
+
+- Contact the Prisma Data Platform Support team and ask about your current Prisma Data Proxy usage.
+- Review notifications emails about the Prisma Data Proxy usage on your project. We send notifications of reaching 50%, 75%, 90%, and 100% of the Data Proxy usage for your project to the email address associated with your GitHub account.
```

Comment:
Notifications are being send manually so I would avoid mentioning specific numbers, e.g. 75% may be sent at 90%, actually.

Reply:
>to the email address associated with your GitHub account.
Let's double check this - i think we discussed sending it more emails, including the billing email and potentially other people on the project.

Vladi should be able to confirm.